cli/io/cbt.py

```diff
@@ -0,0 +1,189 @@
+import copy
+import json
+import re
+from pathlib import Path
+
+import requests
+import yaml
+
+from cli import Cli
+from utility.log import Log
+
+required_packages = [
+    "blktrace",
+    "seekwatcher",
+    "perf",
+    "valgrind",
+    "fio",
+    "cosbench",
+    "pdsh",
+]
+log = Log(__name__)
+
+
+class CbtError(Exception):
+    pass
+
+
+class CBT(Cli):
+    def __init__(self, primary_client):
+        super(CBT, self).__init__(primary_client)
+        self.primary_client = primary_client
+
+        self.required_packages = [
+            "blktrace",
+            "seekwatcher",
+            "perf",
+            "valgrind",
+            "fio",
+            "cosbench",
+            "pdsh",
+            "pdsh-rcmd-ssh",
+        ]
+        self.cbt_conf = """cluster:
+          user: 'cephuser'
+          head: "{head_placeholder}"
+          clients: ["{client_placeholder}"]
+          osds: {osd_nodes}
+          mons:
+            {mon_placeholder}:
+              a: "{ip_placeholder}:{port_placeholder}"
+          osds_per_node: 1
+          fs: 'xfs'
+          mkfs_opts: '-f -i size=2048'
+          mount_opts: '-o inode64,noatime,logbsize=256k'
+          conf_file: '/etc/ceph/ceph.conf'
+          rbd_cmd: 'sudo /usr/bin/rbd'
+          ceph_cmd: 'sudo /usr/bin/ceph'
+          iterations: 1
+          use_existing: True
+          clusterid: "ceph"
+          tmp_dir: "/tmp/cbt"
+          pool_profiles:
+            rbd:
+              pg_size: 256
+              pgp_size: 256
+              replication: 3
+        """
```

Comment:
IMHO, keep the configuration in a separate file. Then read/load the configuration. 

Sometime down the road, we can allow custom configurations to be provide via the test module.